src/script/vco-ie/main_warhammer/wh3_main_ksl_the_great_orthodoxy/missions.lua

```diff
@@ -7,13 +7,19 @@ local missions = {
 			issuer CLAN_ELDERS;
 			primary_objectives_and_payload
 			{
-
+
+
				objective
+				{
+					override_text mission_text_text_vco_ksl_ort_1_there_can_only_be_one;
+					type SCRIPTED;
+					script_key vco_dummy;
+				}
+
 				objective
 				{
-					override_text mission_text_text_vco_ksl_ort_1_there_can_only_be_one;
 					type CONTROL_N_REGIONS_INCLUDING;
 					total 3;
-

```

Comment:
This makes me feel this is a CRLF vs LF thing @Mjb141. Changes detected even when there are no changes.